Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM ghcr.io/linuxserver/baseimage-alpine:3.16
+
+# set version label
+ARG BUILD_DATE
+ARG VERSION
+LABEL build_version="Linuxserver.io version:- ${VERSION} Build-date:- ${BUILD_DATE}"
+LABEL maintainer="nomandera,nemchik"
+
+# environment settings
+ENV S6_BEHAVIOUR_IF_STAGE2_FAILS=2
+
+RUN \
+  echo "**** install runtime packages ****" && \
+  apk add --no-cache --upgrade \
+    curl \
+    fail2ban \
+    jq \
+    nftables && \
+  echo "**** copy fail2ban confs to /defaults ****" && \
+  mkdir -p \
+    /tmp/fail2ban-confs \
+    /defaults/fail2ban/action.d \
+    /defaults/fail2ban/filter.d \
+    /defaults/fail2ban/jail.d && \
+  curl -o \
+    /tmp/fail2ban-confs.tar.gz -L \
+    "https://github.com/linuxserver/fail2ban-confs/tarball/master" && \
+  tar xf \
```

Comment:
Can you not just untar directly into the defaults directory? It looks like you're just copying 1:1 anyway.